### ISSUE TITLE
[Adaptive Cards-Host Config]: Incorrect tooltip as 'Open Settings (Ctrl+,)' is defined for the 'Host Config' control present on the header section of the application.

### ISSUE THREAD

**Issue by Rishikesh6521** [microsoft/AdaptiveCards#9213]
### Target Platforms

UWP

### SDK Version

1.0.3.0

### Application Name

Adaptive Cards

### Problem Description

**Test Environment:**
Application: Adaptive Cards Visualizer
OS Version: 25H2 (OS Build 26200.5670) 
App Version: 1.0.3.0

**Pre-requisites:**
Navigate using keyboard only.

**Repro steps:**
1. Open the Adaptive Cards Visualizer application.
2. Navigate to the 'Host Config' control present on the header section of the application using tab key.
3. Verify that, incorrect tooltip as 'Open Settings (Ctrl+,)' is defined for the 'Host Config' control present on the header section of the application or not.

**Actual Result:**
Incorrect tooltip as 'Open Settings (Ctrl+,)' is defined for the 'Host Config' control present on the header section of the application.

**Expected Result:**
Proper descriptive and relevant tooltip such as 'Open Host Config(Ctrl+,)' should be defined for the 'Host Config' control present on the header section of the application.

**User Impact:**
Users with visual or cognitive disabilities who face difficulty in perceiving the sensory information like shape, size, location, additional information, etc. due to which users will be impacted if and might not be able to identify the proper purpose of controls if tooltip provides incorrect information. 

Have feedback to share on Bugs? please tag bug as “A11yRCA” and add your feedback in the comment section

### Screenshots

https://github.com/user-attachments/assets/a3449795-4943-43cb-92fc-33a3cb4ef92d

### Card JSON

```JSON
N/A
```
